openlibrary/solr/updater/work.py

```diff
@@ -315,11 +315,12 @@ def subtitle(self) -> str | None:
 
     @property
     def alternative_title(self) -> set[str]:
-        return {
-            title
-            for book in (EditionSolrBuilder(self._work), *self._solr_editions)
-            for title in book.alternative_title
-        }
+        alt_title_set = set()
+        for book in (EditionSolrBuilder(self._work), *self._solr_editions):
+            alt_title_set.update(set(book.alternative_title))
```

Comment:
This is already a set
```suggestion
            alt_title_set.update(book.alternative_title)
```